### ISSUE TITLE
Clarify "allowCommands" setup

### ISSUE THREAD

**Issue by eric-burel** [stackblitz/tutorialkit#190]
### Is your feature request related to a problem?

I am hitting an error `jsh: permission denied: astro` when switching from the Vite default template to an Astro app.
After some digging, this seems to be a security system that prevents the terminal to run random commands in the webcontainer. And it seems that the `allowCommands` configuration field should help with that, but I fail to figure it out based on the current documentation.

![image](https://github.com/user-attachments/assets/62a1c1c2-ccd6-453b-bf79-80282b0206be)

### Describe the solution you'd like.

- We may be able to define some default allowed commands at template level, or at tutorial level
- The `allowCommands` setting should be made explicit in the Vite demo so we quickly figure how to tweak it for a new technology (if it's the right setting which I am not sure of yet)
- The permission denied error could be made more explicit
- Tiny improvements : highlight and ctrl+c doesn't seem to work in the terminal, also I observe a freeze just before the webcontainer starts installing package, not sure of the cause yet
- I am pretty sure I didn't observe the same behaviour on my other computer, the Next app was running with no setting from me. I don't have a reliable reproduction yet, perhaps you updated the project since.

### Describe alternatives you've considered.

I've tried setting up my lesson frontmatter like so, no luck:
```
---
type: lesson
title: Welcome to AstroPatterns
terminal:
  panels: "output"
  allowCommands: ["astro"]
```

### Additional context

I think the error is related to allowCommands but perhaps the "jsh" error stems from another issue I don't know about.

**Comment by eric-burel** [comment]
Awesome, closing in favour of #187 
It does have a node_modules indeed, I create new templates by running commands which usually install node_modules too

**Comment by AriPerkkio** [comment]
This is now fixed in `0.1.2` versions. Now `node_modules` of templates are not copied to lesson code. It's safe to have `node_module` present on file system for testing purposes.
